src/TaskRunner.php

```diff
@@ -144,11 +144,38 @@ private function createConfiguration()
             __DIR__.'/../config/runner.yml',
             'runner.yml.dist',
             'runner.yml',
+            $this->getLocalConfigurationFilepath(),
         ], $config);
 
         return $config;
     }
 
+  /**
+   * Get the local configuration filepath.
+   *
+   * @param string $configuration_file
+   *   The default filepath.
+   *
+   * @return string|null
+   *   The local configuration file path, or null if it doesn't exist.
+   */
+    private function getLocalConfigurationFilepath($configuration_file = 'openeuropa/taskrunner/runner.yml')
+    {
```

Comment:
Can you add some documentation concerning these env variables? (```TASKRUNNER_CONFIG```, ```XDG_CONFIG_HOME``` and ```HOME```).

Reply:
Hello,

Here's the explanation:

* TASKRUNNER_CONFIG: Just like consolidation/robo does it with the variable [ROBO_CONFIG](https://github.com/consolidation/Robo/blob/1.x/robo#L40)

* XDG_CONFIG_HOME: This is the standardized way of discovering the location of the config is defined in the [XDG Basedir Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html):
  * First check if the $XDG_CONFIG_HOME environment variable is set. Use it if it is.
  * If the environment variable is not set, use $HOME/.config.
  * If $HOME is not set, don't search for a local config file.

I don't think I should include this in the task-runner documentation, don't you think ?

Reply:
If we are creating a variable, we should provide documentation.
see: https://robo.li/getting-started/ explains ROBO_CONFIG.

Reply:
Fixed by @pfrenssen